resources/migrations/001_update_migrations.yaml

```diff
@@ -5823,6 +5823,60 @@ databaseChangeLog:
         - sql:
             sql: >-
               DROP VIEW IF EXISTS v_content;
+  - changeSet:
+      id: v50.2024-02-29T15:06:43
+      author: tsmacdonald
+      comment: Add the field_usage join table
+      changes:
```

Comment:
@qnkhuat this is our chance to bikeshed about the `FieldUsage` name

Reply:
how about field_ref? or field_reference?